src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_IVariableDeclaration.cs

```diff
@@ -2287,6 +2287,69 @@ static void Main(string[] args)
             VerifyOperationTreeAndDiagnosticsForTest<VariableDeclarationSyntax>(source, expectedOperationTree, expectedDiagnostics);
         }
 
+        [CompilerTrait(CompilerFeature.IOperation)]
+        [Fact]
+        public void UsingStatement_DefaultDisposeArguments()
```

Comment:
>UsingStatement_DefaultDisposeArguments [](start = 20, length = 38)

It feels like we should have some coverage for CFG as well.

Reply:
See UsingFlow_18/19/20

---
In reply to: [539529809](https://github.com/dotnet/roslyn/pull/49842#discussion_r539529809) [](ancestors = 539529809)